popup-script.js

```diff
@@ -0,0 +1,71 @@
+  technologies: ['HTML', 'CSS', 'Javascript'],
+},
+{
+  title: 'Profesional Art Printing Data',
+  image: ['popup-mobile.svg', 'popup-desktop.svg'],
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+},
+{
+  title: 'Profesional Art Printing Data',
+  image: ['popup-mobile.svg', 'popup-desktop.svg'],
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+},
+{
+  title: 'Profesional Art Printing Data',
+  image: ['popup-mobile.svg', 'popup-desktop.svg'],
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+},
+{
+  title: 'Profesional Art Printing Data',
+  image: ['popup-mobile.svg', 'popup-desktop.svg'],
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+},
+{
+  title: 'Profesional Art Printing Data',
+  image: ['popup-mobile.svg', 'popup-desktop.svg'],
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+}];
```

Comment:
- [ ] In the mobile size: When I click on a random project, it displays a popup without content. Kindly work on it.
<img width="212" alt="image" src="https://user-images.githubusercontent.com/77190203/152173773-39699de9-f139-46b7-b2f5-519b8c40424b.png">